docs/content/direct/get-started.md

```diff
@@ -10,11 +10,29 @@ This page shows one concrete example of steps 2--7 from the [full Installation a
   2. [Exercise KubeStellar](#exercise-kubestellar)
   3. [Troubleshooting](#troubleshooting)
 
+
+
 ## Setup
 
 This is one way to produce a very simple system, suitable for study but not production usage. For general setup information, see [the full story](user-guide-intro.md#the-full-story).
 
-### Install software prerequisites
+### Automated Shell Script
+
+There are two ways to run the setup: using our automated shell script or running the commands step-by-step. For a quick and easy setup of a KubeStellar demo environment, you can use the provided script. This script automates the process of creating a KubeStellar environment suitable for demonstration purposes. It performs the following tasks:
+
+1. Checks for required software prerequisites
+2. Cleans up any existing environment from previous runs
+3. Creates a KubeFlex hosting cluster and installs KubeStellar core components
+4. Creates and registers two Work Execution Clusters (WECs)
+
+
+To use this script, run the following command:
+```shell 
+bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/refs/heads/main/scripts/create-kubestellar-demo-env.sh)
```

Comment:
Consider what happens when somebody is looking at the copy of this document in release X but there have been changes since then and the copy of this script in the `main` branch behaves differently. Such a reader is going to be confused. Better to have the document in release X refer the reader to the script in release X.